README.md

```diff
@@ -186,7 +186,9 @@ The unit tests are also built when the solution is compiled. For information on
 The docker-compose-standalone.yml file is meant for local testing/troubleshooting.
 
 To utilize this, pass the -f flag to the docker-compose command as follows:
-> docker-compose -f docker-compose-confluent-cloud.yml up
+> docker-compose -f docker-compose-standalone.yml up
+
+Sometimes kafka will fail to start up properly. If this happens, spin down the containers and try again.
```

Comment:
Do we know why it failed to start up? Why restart the container resolve the issues?

Reply:
This is the relevant error:
> kafka_1      | [2023-03-06 18:58:30,514] ERROR Error while creating ephemeral at /brokers/ids/1001, node already exists and owner '72070068176224256' does not match current session '72070094776041472' (kafka.zk.KafkaZkClient$CheckedEphemeral)
kafka_1      | [2023-03-06 18:58:30,521] ERROR [KafkaServer id=1001] Fatal error during KafkaServer startup. Prepare to shutdown (kafka.server.KafkaServer)
kafka_1      | org.apache.zookeeper.KeeperException$NodeExistsException: KeeperErrorCode = NodeExists
kafka_1      |  at org.apache.zookeeper.KeeperException.create(KeeperException.java:126)
kafka_1      |  at kafka.zk.KafkaZkClient$CheckedEphemeral.getAfterNodeExists(KafkaZkClient.scala:1904)
kafka_1      |  at kafka.zk.KafkaZkClient$CheckedEphemeral.create(KafkaZkClient.scala:1842)
kafka_1      |  at kafka.zk.KafkaZkClient.checkedEphemeralCreate(KafkaZkClient.scala:1809)
kafka_1      |  at kafka.zk.KafkaZkClient.registerBroker(KafkaZkClient.scala:96)
kafka_1      |  at kafka.server.KafkaServer.startup(KafkaServer.scala:308)
kafka_1      |  at kafka.Kafka$.main(Kafka.scala:109)
kafka_1      |  at kafka.Kafka.main(Kafka.scala)

It looks like a broker already exists when attempting to create it. Spinning down the containers w/ `docker-compose down` should remove the existing broker.